configure.ac

```diff
@@ -1635,6 +1635,12 @@ SQUID_CHECK_LIB_WORKS(ldap,[
     AC_CHECK_HEADERS(mozldap/ldap.h)
     SQUID_CHECK_LDAP_API
   ])
+  # We can still rely on deprecated APIs on MacOS
+])
```

Comment:
This comment looks misplaced to me.

```suggestion
])
# We can still rely on deprecated APIs on MacOS
```

Personally, I would even lower this comment one more line, placing it inside the AS_IF() statement.

Reply:
removed